contracts/consensus-gateway/ConsensusCogateway.sol

```diff
@@ -324,10 +319,7 @@ contract ConsensusCogateway is MasterCopyNonUpgradable, MessageBus, ConsensusGat
             msg.sender
         );
 
-        require(
-            ERC20I(most).burnFrom(msg.sender, _amount),
-            "Utmost burnFrom must succeed."
-        );
+        MOSTInterface(address(most)).burnFrom(msg.sender, _amount);
```

Comment:
This is auxiliary chain side. 
We can use `UtilityTokenInterface`


I am adding a point for discussion that the variable most, must be renamed as `baseToken` to make it generic, no change is required for this rename in this ticket.

Reply:
Done